client/state/plugins/premium/selectors.js

```diff
@@ -0,0 +1,54 @@
+import find from 'lodash/find';
+import filter from 'lodash/filter';
+
+const isRequesting = function( state, siteId ) {
+	// if the `isRequesting` attribute doesn't exist yet,
+	// we assume we are still launching the fetch action, so it's true
+	if ( typeof state[ siteId ] === 'undefined' ) {
+		return true;
+	}
+	return state[ siteId ];
```

Comment:
Our current convention is to pass through the full state object to our selectors. We need to tidy up the interfaces of the other existing plugin selectors, in a future PR. See also [selectors](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#selectors)

So this might look like:
```javascript
return state.plugins.premium[ stateId ];
```